docs/content/tutorial/saving-your-data.mdx

```diff
@@ -150,15 +263,18 @@ def topstories(topstory_ids):
         if len(results) % 20 == 0:
             logger.info(f"Got {len(results)} items so far.")
 
-    df = pd.DataFrame(results).drop(["kids"], axis=1)
+    df = pd.DataFrame(results).drop(
+        ["kids"], axis=1
+    )  # TODO - why are we dropping kids column here, but not in the other versions of this asset?
```

Comment:
resolve this TODO - cc @erinkcochran87  maybe you have some context as to why`.drop(["kids"], axis=1) is added in here?

Reply:
Unfortunately I don't - I honestly think this was very likely an oversight and not intentional.